app/src/main/java/org/greenstand/android/TreeTracker/fragments/NewTreeFragment.kt

```diff
@@ -85,6 +88,11 @@ class NewTreeFragment : androidx.fragment.app.Fragment(), OnClickListener, Activ
 
         mImageView = v.fragmentNewTreeImage
 
+        frameLayout = v.fragment_new_tree_frame_layout
+
+        focusWarning = v.fragment_new_tree_focus_warning_text

```

Comment:
These local references to the view are not needed. We can directly use fragment_new_tree_frame_layout and fragment_new_tree_focus_warning_text without using `v`

Reply:
Ok this is done.